exchanges/bybit/bybit_wrapper.go

```diff
@@ -1959,7 +1976,78 @@ func (by *Bybit) GetLatestFundingRates(ctx context.Context, r *fundingrate.Lates
 }
 
 // GetOpenInterest returns the open interest rate for a given asset pair
-func (by *Bybit) GetOpenInterest(context.Context, ...key.PairAsset) ([]futures.OpenInterest, error) {
-	// TODO: implement with v5 API upgrade
-	return nil, common.ErrNotYetImplemented
+func (by *Bybit) GetOpenInterest(ctx context.Context, k ...key.PairAsset) ([]futures.OpenInterest, error) {
+	by.Verbose = true
+	for i := range k {
+		if k[i].Asset != asset.USDCMarginedFutures &&
+			k[i].Asset != asset.USDTMarginedFutures &&
+			k[i].Asset != asset.CoinMarginedFutures {
+			return nil, fmt.Errorf("%w %v", asset.ErrNotSupported, k[i].Asset)
+		}
+	}
+	if resp, err := by.GetCachedOpenInterest(ctx, k...); err == nil && len(resp) > 0 {
+		return resp, nil
+	}
```

Comment:
The only issue with caching is that if somebody just uses the exchange package and they call update tickers, once or every hour this could be not as exact as is expected and if you are trying to size deployment from it you could under commit or skew the books. 
Suggestion: We just RM this and do the rest request. Up to you.

Reply:
I'll add an additional bool for you to ignore it since you've brought this up twice. I want this feature as it saves me on requests

Reply:
It has too many annoyances to change the signature. I can still work with it though the base implementation